### PR TITLE
Add review badge when all jobs passed or reviewed

### PR DESCRIPTION
**Pull request by Martchus**
Implement optional task in https://progress.opensuse.org/issues/14934

### CODE REVIEW

File: lib/OpenQA/BuildResults.pm [hunk 1]
```diff
@@ -68,6 +68,8 @@ sub add_review_badge {
     my ($build_res) = @_;
 
     $build_res->{reviewed_all_passed} = $build_res->{passed} == $build_res->{total};
+    $build_res->{reviewed_all_passed_or_softfailed}
+      = $build_res->{passed} + $build_res->{softfailed} == $build_res->{total};
```
Comment (Martchus) [inline lib/OpenQA/BuildResults.pm]: Think about it as "auto-review" badge. But I can change it if you like. Then I would also change `reviewed_all_passed` for consistency.

Comment (coolo) [inline lib/OpenQA/BuildResults.pm]: I disagree that passed tests are auto-reviewed. They are not reviewed at all

Comment (Martchus) [inline lib/OpenQA/BuildResults.pm]: Changed it

**Comment by coolo** [comment]
fine with me

**Pull request merged**